dev/dune-workspace.all

```diff
@@ -0,0 +1,14 @@
+(lang dune 1.2)
+
+; Add custom flags here. Default developer profile is `dev`
+(env
+ (dev     (flags :standard -rectypes -w -9-27-50))
+ (release (flags :standard -rectypes)))
+
+(context (opam (switch 4.02.3)))
+(context (opam (switch 4.02.3+32bit)))
+; (context (opam (switch 4.05.0)))
+; (context (opam (switch 4.05.0+32bit)))
```

Comment:
Why is this commented? And isn't 4.06 missing?

Reply:
It was commented to reflect the already done move. I've amended the commit message to indicate this file compiles Coq for all versions that are tested in the CI, not for all the supported ones, which are IMO too many.

Of course one can just add more versions there.